README.md

```diff
@@ -46,35 +46,88 @@ application you can leave away the `poetry run` part of the commands given below
 
 1. To launch the program run `poetry run python -m app_webcam` in the root of this repository
 2. Enjoy
-3. Press `q` to terminate the program
+3. Press `q` or simply click the close button to terminate the program
```

Comment:
if you add the comment about the close button here then please also add it on line 42

Reply:
👍